src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -425,41 +425,17 @@ namespace winrt::Microsoft::Terminal::Control::implementation
             return;
         }
 
-        auto newOpacity = std::clamp(_settings.TintOpacity() + adjustment,
+        auto newOpacity = std::clamp(_settings.Opacity() + adjustment,
                                      0.0,
                                      1.0);
-        if (_settings.UseAcrylic())
-        {
-            try
-            {
-                _settings.TintOpacity(newOpacity);
 
-                if (newOpacity >= 1.0)
-                {
-                    _settings.UseAcrylic(false);
-                }
-                else
-                {
-                    // GH#5098: Inform the engine of the new opacity of the default text background.
-                    SetBackgroundOpacity(::base::saturated_cast<float>(newOpacity));
-                }
+        // GH#5098: Inform the engine of the new opacity of the default text background.
+        SetBackgroundOpacity(::base::saturated_cast<float>(newOpacity));
 
-                auto eventArgs = winrt::make_self<TransparencyChangedEventArgs>(newOpacity);
-                _TransparencyChangedHandlers(*this, *eventArgs);
-            }
-            CATCH_LOG();
-        }
-        else if (adjustment < 0)
-        {
-            _settings.UseAcrylic(true);

```

Comment:
If I had to guess, this is part of what breaks the "go from solid to acrylic" wheel action on Windows 10. ☹️

Reply:
Oooof. Might need to add this back to fix #11285, to make sure that we have some acrylic on win10. Gross.